spec/models/like_spec.rb

```diff
@@ -0,0 +1,11 @@
+require 'rails_helper'
+
+RSpec.describe Like, type: :model do
+  subject { Like.new(author_id: 3, post_id: 3) }
+
+  context 'it should exist' do
+    it 'should exist as instance' do
+      expect(subject).to be_instance_of Like
+    end
+  end
+end
```

Comment:
- [ ] kindly add test for the method in your like mode class `updatelikescounter`

Reply:
@VICTOR-CHUKWU  done